api/webhook.js

```
@@ -0,0 +1,250 @@
+            const sessionInfo = activeSessions.get(userId);
+            const duration = sessionInfo ? Math.floor((Date.now() - sessionInfo.startTime) / 1000) : 0;
+            
+            return `🎯 ═══════════════════════════════════════ 🎯\n\n📊 STATUT DU SYSTÈME RP\n\n🔥 État: ${isActive ? '✅ ACTIF' : '❌ INACTIF'}\n🌍 Monde: ${config.rpWorld}\n🤖 Personnage: ${config.defaultCharacter}\n📸 Médias: ${config.enableMediaResponses ? '✅ Activés' : '❌ Désactivés'}\n⏱️ Durée: ${duration}s\n\n⚡ ═══════════════════════════════════════ ⚡`;
+            
+        case 'help':
+            const commandList = Object.entries(rpCommands)
+                .map(([cmd, desc]) => `🎮 ${config.commandPrefix} ${cmd} - ${desc}`)
+                .join('\n');
+                
+            return `🎮 ═══════════════════════════════════════ 🎮\n\n📋 COMMANDES DISPONIBLES\n\n${commandList}\n\n✨ FONCTIONNALITÉS ✨\n\n🤖 IA Gemini pour réponses immersives\n📸 Médias automatiques après actions\n🎨 Caractères spéciaux et emojis\n🌃 Univers ${config.rpWorld}\n\n🔮 ═══════════════════════════════════════ 🔮`;
+            
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)